raspi/README.md

```diff
@@ -0,0 +1,232 @@
+    │ └─ubuntu--vg-root   253:0    0 229.6G  0 lvm  /
+    └─sda1                  8:1    0   512M  0 part /boot/efi
+
+Next, mount the first slot (called a *partition*) on your system:
+
+    $ sudo mount /dev/sdb1 /media
+
+Take a look at the files:
+
+    $ ls /media
+    bcm2708-rpi-0-w.dtb     bcm2710-rpi-3-b.dtb  COPYING.linux  issue.txt         overlays
+    bcm2708-rpi-b.dtb       bcm2710-rpi-cm3.dtb  fixup_cd.dat   kernel7.img       start_cd.elf
+    bcm2708-rpi-b-plus.dtb  bootcode.bin         fixup.dat      kernel.img        start_db.elf
+    bcm2708-rpi-cm.dtb      cmdline.txt          fixup_db.dat   LICENCE.broadcom  start.elf
+    bcm2709-rpi-2-b.dtb     config.txt           fixup_x.dat    LICENSE.oracle    start_x.elf
+
+We first configure the wireless networking by creating a file in this directory,
+`wpa_supplicant.conf`, that should contain this (yeah, you can create/edit this
+file with your regular programming editors, like Atom):
+
+    ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
+    update_config=1
+    country=US
+
+    network={
+        ssid="«your_SSID»"
+        psk="«your_PSK»"
+        key_mgmt=WPA-PSK
+    }
+
+Remember to change the `ssid` line with the name of the Wireless network to
+connect, and `psk` is its password. While you are at it, create an empty file in
+this directory called `ssh`.
+
+When you are done, un-mount the card with the following Terminal command:
+
+    $ sudo umount /media
+
+Take your MicroSD card from the reader, and back into a Raspberry Pi. When you
+power it on, it will connect to your network, and allow you to log into it over
+the network. The problem is, you need to know its *numeric networking address*.
+
+The Adafruit company has made a nice program called [Pi Finder][pif]. This
+program scans the network looking for Raspberry Pi, and allows you to open a
+**Terminal** on the Pi (assuming you created the `ssh` file mentioned above) and
+**Upload** files from your computer to it. Click the **Bootstrap** button to
+install some programs (see [these details][pf2]).
+
+  [pif]: https://github.com/adafruit/Adafruit-Pi-Finder
+  [pf2]: https://learn.adafruit.com/the-adafruit-raspberry-pi-finder/overview
+
+Install our software project on the system
+------------------------------------------
+
+Using either the **Terminal** button on the *Pi Finder* application, or starting
+up the **Terminal** on your laptop and typing:
+
+    $ ssh pi@10.0.1.187  # Or whatever the IP address is
+
+Once you are logged into the system, let's first secure the system by changing the password:
+
+    $ passwd
+    Changing password for pi.
+    (current) UNIX password:       # This is probably raspberry
+    Enter new UNIX password:       # Set to pigmice2733
+    Retype new UNIX password:
+    passwd: password updated successfully
+
+Next, let's install `git` and the other dependencies:
+
+    $ sudo apt-get install -y git python-pip python-opencv
```

Comment:
Remember how the `python-opencv` installs an older version of OpenCV, and we needed to follow these instructions in order to build it from scratch?

https://www.pyimagesearch.com/2017/09/04/raspbian-stretch-install-opencv-3-python-on-your-raspberry-pi/